README.md

```diff
@@ -1,75 +1,93 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Doctors Appointment Backend
```

Comment:
- [ ]  Your API does not have any documentation. Please add documentation about how to consume your API:

![https://user-images.githubusercontent.com/57421823/200048389-b9654251-91e3-4642-9cd5-c65f71a11960.png](https://user-images.githubusercontent.com/57421823/200048389-b9654251-91e3-4642-9cd5-c65f71a11960.png)

You could use [rswag](https://github.com/rswag/rswag) which will add the documentation and tests at the same time 💪